lib/videoClient.ts

```diff
@@ -107,79 +112,103 @@ const ZoomVideo = (credential): VideoApiAdapter => {
-      }
-    }).then(handleErrorsRaw)),
-    updateMeeting: (uid: String, event: CalendarEvent) => auth.getToken().then(accessToken => fetch('https://api.zoom.us/v2/meetings/' + uid, {
-      method: 'PATCH',
-      headers: {
-        'Authorization': 'Bearer ' + accessToken,
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(translateEvent(event))
-    }).then(handleErrorsRaw)),
-  }
+    createMeeting: (event: CalendarEvent) =>
+      auth.getToken().then((accessToken) =>
+        fetch("https://api.zoom.us/v2/users/me/meetings", {
+          method: "POST",
+          headers: {
+            Authorization: "Bearer " + accessToken,
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(translateEvent(event)),
+        }).then(handleErrorsJson)
+      ),
+    deleteMeeting: (uid: string) =>
+      auth.getToken().then((accessToken) =>
+        fetch("https://api.zoom.us/v2/meetings/" + uid, {
+          method: "DELETE",
+          headers: {
+            Authorization: "Bearer " + accessToken,
+          },
+        }).then(handleErrorsRaw)
+      ),
+    updateMeeting: (uid: string, event: CalendarEvent) =>
+      auth.getToken().then((accessToken) =>
+        fetch("https://api.zoom.us/v2/meetings/" + uid, {
+          method: "PATCH",
+          headers: {
+            Authorization: "Bearer " + accessToken,
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(translateEvent(event)),
+        }).then(handleErrorsRaw)
+      ),
+  };
 };
 
 // factory
-const videoIntegrations = (withCredentials): VideoApiAdapter[] => withCredentials.map((cred) => {
-  switch (cred.type) {
-    case 'zoom_video':
-      return ZoomVideo(cred);
-    default:
-      return; // unknown credential, could be legacy? In any case, ignore
-  }
-}).filter(Boolean);
-
-
-const getBusyVideoTimes = (withCredentials, dateFrom, dateTo) => Promise.all(
-  videoIntegrations(withCredentials).map(c => c.getAvailability(dateFrom, dateTo))
-).then(
-  (results) => results.reduce((acc, availability) => acc.concat(availability), [])
-);
-
-const createMeeting = async (credential, calEvent: CalendarEvent): Promise<any> => {
+const videoIntegrations = (withCredentials): VideoApiAdapter[] =>
+  withCredentials
+    .map((cred) => {
+      switch (cred.type) {
+        case "zoom_video":
+          return ZoomVideo(cred);
+        default:
+          return; // unknown credential, could be legacy? In any case, ignore
+      }
+    })
+    .filter(Boolean);
+
+const getBusyVideoTimes = (withCredentials, dateFrom, dateTo) =>
+  Promise.all(videoIntegrations(withCredentials).map((c) => c.getAvailability(dateFrom, dateTo))).then(
+    (results) => results.reduce((acc, availability) => acc.concat(availability), [])
+  );
+
+const createMeeting = async (
+  credential,
+  calEvent: CalendarEvent,
+  emailTemplates: EmailTemplate[]
+): Promise<any> => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689304)
```suggestion
): Promise<unknown> => {
```